ccextender.yaml

```diff
@@ -1,44 +1,316 @@
+        - library
+    both-io:
+      - input
+      - output
+      - streamer-manager
+    input:
+      - input
+      - streamer-manager
+    output:
+      - output
+      - streamer-manager
 change-packs:
-  docker:
-    template-go-docker:
-  go-service:
-    template-go-layout:
-    template-go-lint:
-  oss:
-    template-oss:
+  golang-base:
+    template-base:
+    template-go-base:
+  python-base:
+    template-base:
+    template-python-base:
+  library:
+  service:
+  golang-open-library:
+    # Currently nothing needed here
+  python-open-library:
+    # Currently nothing needed here
+  golang-private-library:
+    template-go-private-library:
+  python-private-library:
+    # Currently nothing needed here
+  python-open-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-open-service:
+    template-docker:
+  python-private-service:
+    template-makefile:
+        sdcli_lang: python
+    template-python-private-service:
+  golang-open-service:
+    template-go-open-service:
+    template-docker:
+    template-makefile:
+      sdcli_lang: go
+  golang-private-service:
+    template-go-private-service:
+    template-makefile:
+      sdcli_lang: go
+  input:
+    template-inbound:
+    template-micros:
+      service_descriptor: "
+                           \ benthos-input:\n
+                           \   image: ${BENTHOS_INPUT_ARTIFACT_NAME} #docker.atl-paas.net/sox/asecurityteam/sec-nexpose-asset-producer-benthos-input\n
+                           \   tag: ${ARTIFACT_VERSION}\n
+                           \   deployRestartCount: 3\n
+                           \   ports:\n
+                           \     - \"4195:4195\"\n
+                           \   links:\n
+                           \     - gateway-incoming-private\n
+                           \     - statsd-service-sidecar\n
+                           \ gateway-incoming-private:\n
+                           \   image: ${GATEWAY_INCOMING_PRIVATE_ARTIFACT_NAME} #docker.atl-paas.net/sox/asecurityteam/sec-awsconfig-filterd-gateway-incoming-private\n
+                           \   tag: ${ARTIFACT_VERSION}\n
+                           \   deployRestartCount: 3\n
+                           \   links:\n
+                           \     - app\n
+                           \     - statsd-service-sidecar"
+      docker_compose: "
+                       \ benthos-input:\n
+                       \   build:\n
+                       \     context: .\n
+                       \     dockerfile: benthos-input.Dockerfile\n
+                       \     args:\n
+                       \       REGISTRY: ${REGISTRY}\n
+                       \   image: ${IMAGE}-benthos-input:${TAG}\n
+                       \   environment:\n
+                       \     APP_INPUT_ENDPOINT: \"http://gateway-inbound-private:8082/event\"\n
+                       \     PORTS_BENTHOS_INPUT_STREAMS: \"4195\"\n
+                       \     SQS_CONFIG_QUEUE_URL: \"\"\n
```

Comment:
Suggestion: I think this variable would be more clear if it were called `SQS_INPUT_QUEUE_URL` instead of `SQS_CONFIG_QUEUE_URL`. It would match the `SQS_OUTPUT_QUEUE_URL` that's specified below a little better too.

Reply:
Good point! Will make that change soon.